pkg/venafi/firefly/connector.go

```diff
@@ -25,6 +25,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/Venafi/vcert/v5/pkg/util"
```

Comment:
Sort this import accordingly to our rules

Reply:
It's done